wagtail/admin/templates/wagtailadmin/shared/breadcrumbs.html

```diff
@@ -62,10 +63,16 @@
                                 {% if trailing_breadcrumb_title or is_expanded %}data-breadcrumb-item{% endif %}
                                 {% if trailing_breadcrumb_title and not is_expanded %}hidden{% endif %}
                             >
-                                <a class="{{ breadcrumb_link_classes }}"
-                                    href="{{ breadcrumb_url }}{{ querystring_value }}">
-                                    {{ page.get_admin_display_title }}
-                                </a>
+                                {% if unlink_last %}
+                                    <span class="{{ breadcrumb_link_classes }} w-breadcrumb-last hover:w-no-underline">
+                                        {{ page.get_admin_display_title }}
```

Comment:
Just checking - is the `trailing_breadcrumb_title` that this tag already supports sufficient here?
It is a non-link (div) element that sits at the end of the breadcrumbs.

As per the description above
> `trailing_breadcrumb_title` (string?) - use this for a non linkable last breadcrumb